README.md

```diff
@@ -858,6 +859,32 @@ Iterates over a list of elements, yielding each in turn to an iteratee function.
 
 **[⬆ back to top](#quick-links)**
 
+### _.each(object)
+
+Iterates over all enumerable string-keyed properties of an object, skipping properties in the prototype chain, yielding each in turn to an iteratee function.
+
+  ```js
+  // Underscore/Lodash
+  _.each({ 'a': 1, 'b': 2, 'c': 3 }, function(value, key) {
+    console.log(key);
+  });
+  // output: a b c
+
+  // Native
+  Object.entries({ 'a': 1, 'b': 2, 'c': 3 }).forEach(function ([key, value]) {
```

Comment:
Hmmm... I wonder, wouldn't be better to use forin loop? It has support for more browsers than `Object.entries`

Reply:
Well, perhaps the same argument could be made for arrays too (for-loop vs .forEach)...

Could the following be a compromise?
```
var obj = { 'a': 1, 'b': 2, 'c': 3 };
Object.keys(obj).forEach(function(key) {
  var value = obj[key];
  console.log(key, value);
});
```
^ IE9+ supported them, in line with the current suggested array solution.